File: wakame-plugin/src/main/java/cc/mewcraft/wakame/event/NekoLoadDataEvent.java
package cc.mewcraft.wakame.event;

import org.bukkit.event.Event;
import org.bukkit.event.HandlerList;
import org.jetbrains.annotations.NotNull;

public class NekoLoadDataEvent extends Event {

    private static final @NotNull HandlerList HANDLERS = new HandlerList();

    public NekoLoadDataEvent() {
        super(true);
    }

    @Override
    public @NotNull HandlerList getHandlers() {
        return HANDLERS;
    }

    public static @NotNull HandlerList getHandlerList() {
        return HANDLERS;
    }

}

File: wakame-plugin/src/main/kotlin/cc/mewcraft/wakame/event/NekoLoadDataEvent.kt
package cc.mewcraft.wakame.event

import org.bukkit.event.Event
import org.bukkit.event.HandlerList

class NekoLoadDataEvent : Event() {
    override fun getHandlers(): HandlerList = HANDLER_LIST

    companion object {
        @JvmStatic
        val HANDLER_LIST = HandlerList()

        @JvmStatic
        fun getHandlerList(): HandlerList = HANDLER_LIST
    }
}